moto/cloudformation/responses.py

```diff
@@ -818,6 +818,7 @@ def set_stack_policy(self) -> str:
           <member>
             <OutputKey>{{ output.key }}</OutputKey>
             <OutputValue>{{ output.value }}</OutputValue>
+            <Description>{{ output.description }}</Description>
```

Comment:
```suggestion
            {% if output.description %}<Description>{{ output.description }}</Description>{% endif %}
```
Nitpick, but AWS only sends this tag if there is an actual value.

Looks like there's some existing tests failing as well.